roles/test/headbeh.role

```diff
@@ -21,6 +21,8 @@ nuclear_role(
   vision::GreenHorizonDetector
   vision::GoalDetector
   vision::BallDetector
+  # Director
+  # extension::Director
```

Comment:
revert

Reply:
I was leaving these in deliberately for later but all good